monitoring/monitoring/prune.go

```diff
@@ -6,17 +6,17 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/inconshreveable/log15"
+	"github.com/sourcegraph/log"
 
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
-func pruneAssets(logger log15.Logger, filelist []string, grafanaDir, promDir string) error {
+func pruneAssets(logger log.Logger, filelist []string, grafanaDir, promDir string) error {
 	// Prune Grafana assets
 	if grafanaDir != "" {
-		logger.Info("Pruning Grafana assets", "dir", grafanaDir)
+		logger.Info("Pruning Grafana assets", log.String("dir", grafanaDir))
 		err := filepath.Walk(grafanaDir, func(path string, info fs.FileInfo, err error) error {
-			plog := logger.New("path", path)
+			plog := logger.Scoped("path", path)
```

Comment:
See above comments and the docstrings on `Scoped` :)

```suggestion
			plog := logger.With("path", path)
```

Reply:
Also refer to the "How to add logging" docs about using sub-loggers: https://docs.sourcegraph.com/dev/how-to/add_logging#sub-loggers

Reply:
would something like the following be appropriate  :

`plog := logger.Scoped("prune","pruning assets").With(log.String("path", path))`

?

Reply:
I think the scope should be created at a higher level - in the iteration, simply logging the variable being dealt with (`path`) is sufficient :)